src/container/__tests__/FluxContainer-test.js

```diff
@@ -238,7 +238,38 @@ describe('FluxContainer', () => {
     expect(getValue()).toBe('bar');
   });
 
-  it('should get access to context', () => {
+  it('should get access to context in getStores', () => {
+    // Setup the container.
+    class SimpleContainer extends BaseContainer {
+      static contextTypes = {
+        value: React.PropTypes.array,
```

Comment:
Let's make value the particular store for a more realistic example. This will probably need to then be `any` like you did above